tests/test_data/donnahay.com.au/donnahay_2.json

```diff
@@ -0,0 +1,70 @@
+{
+  "host": "donnahay.com.au",
+  "canonical_url": "https://www.donnahay.com.au/recipes/snacks-and-sides/smoky-eggplant-dip-with-hand-cut-potato-chips",
+  "site_name": "Donna Hay",
+  "author": "Donna Hay",
+  "language": "en",
+  "title": "BABA GHANOUSH",
```

Comment:
with the new title method this would fail, the title needs to be in lowercase

Reply:
Makes sense - I'll modify this when the new tests get merged, just to make sure there's no other changes.